### ISSUE TITLE
Dynamic tag not updated after scheduled_auto_updates request

### ISSUE THREAD

**Issue by HarukaIzumisawa** [GoogleCloudPlatform/datacatalog-tag-engine#122]
Summary: 
This issue reports that dynamic tags are not being updated after the [`scheduled_auto_updates`](https://github.com/GoogleCloudPlatform/datacatalog-tag-engine/blob/b64520a4780e66e72215efd5cf530e3f87b61059/main.py#L3047) function is called, even though the next_run parameter has passed. The potential cause might be related to the `scheduling_status` parameter not transitioning to the expected `READY` state.

Details:
We expect that creating a [`Dynamic Tag Table`](https://github.com/GoogleCloudPlatform/datacatalog-tag-engine/blob/b64520a4780e66e72215efd5cf530e3f87b61059/docs/ui_guide.md#-dynamic-tag-table) configuration with the `Refresh Mode` set to `AUTO` and then calling `scheduled_auto_updates` would update the tags based on the configuration. However, no tag updates were implemented.

The reason for this is that the `scheduling_status` parameter does not transition to the expected `READY` state.

The reason for this is as follows.

Looking at the source code, it says that the `scheduling_status` parameter must be `READY` as a condition for updating.

https://github.com/GoogleCloudPlatform/datacatalog-tag-engine/blob/b64520a4780e66e72215efd5cf530e3f87b61059/TagEngineStoreHandler.py#L1340-L1349

However, looking at the source code, if I create a `Dynamic Tag Table` configuration with `Refresh Mode` set to `AUTO`, the `scheduling_status` is set to `ACTIVE`. In fact, the tags were not updated when I ran the `scheduled_auto_updates` function.

https://github.com/GoogleCloudPlatform/datacatalog-tag-engine/blob/b64520a4780e66e72215efd5cf530e3f87b61059/TagEngineStoreHandler.py#L495-L499

If there are additional steps required for the update to happen with `Refresh Mode` set to `AUTO`, we would appreciate clarification.

Call to action:
- Please advise if there are additional steps required for the update to occur with `Refresh Mode` set to `AUTO`.
- If this is a bug, please advise on the next steps or potential workarounds.

**Comment by shirleycohen** [comment]
Thanks for letting me know, this sounds like a bug. Out of curiosity, how were you calling `/scheduled_auto_updates`? Was it from Cloud Scheduler?

**Comment by HarukaIzumisawa** [comment]
With the environment variables set, I used the following command to call `/scheduled_auto_updates`. I didn't use Cloud Scheduler, because I considered that if a direct call with the `curl` command doesn't work, it won't work with cloud scheduler.
```
curl -i -X POST $TAG_ENGINE_URL/scheduled_auto_updates \
  -H "Authorization: Bearer $IAM_TOKEN" \
  -d @examples/configs/dynamic_table/dynamic_table_auto.json
```

**Comment by shirleycohen** [comment]
@HarukaIzumisawa apologies for the long delay. This issue has been fixed in [v2.2.9](https://github.com/GoogleCloudPlatform/datacatalog-tag-engine/releases/tag/v2.2.9)

Please note that you need to call the `scheduled_auto_updates` method as follows:

```
export IAM_TOKEN=$(gcloud auth print-identity-token)

curl -X POST $TAG_ENGINE_URL/scheduled_auto_updates \
  -H "Authorization: Bearer $IAM_TOKEN"
```

The expected output looks similar to:
```
curl -X POST $TAG_ENGINE_URL/scheduled_auto_updates  \
  -H "Authorization: Bearer $IAM_TOKEN"
{"job_ids":"[\"20380bf6340d11ef8ea942004e494300\"]","success":true}
```
This means that Tag Engine found one config that needed to be run and created a job for it. 

When no ready configs are found, the expected output looks like this instead:
```
curl -X POST $TAG_ENGINE_URL/scheduled_auto_updates   \
  -H "Authorization: Bearer $IAM_TOKEN"
{"job_ids":"[]","success":true}
```
